BuildingBlocks/src/BuildingBlocks.Infrastructure/EventBus/AzureServiceBus/EventBusAzureServiceBus.cs

```diff
@@ -225,6 +226,6 @@ internal static partial class EventBusAzureServiceBusLogs
         EventId = 146670,
         EventName = "EventBusAzureServiceBus.ErrorWhileProcessingDomainEvent",
         Level = LogLevel.Error,
-        Message = "An error occurred while processing the domain event with id '{domainEventId}'.")]
-    public static partial void ErrorWhileProcessingDomainEvent(this ILogger logger, string domainEventId, Exception ex);
+        Message = "An error occurred while processing the '{domainEventName}'.")]
```

Comment:
```suggestion
        Message = "An error occurred while processing a '{domainEventName}'.")]
```

Reply:
We discussed the definite and indefinite article topic above, I will leave it as it is.